src/types/form_logic.ts

```diff
@@ -116,4 +111,4 @@ export type LogicCondition =
 /**
  * Logic POJO with functions removed
  */
-export type LogicDto = ConditionalPick<LeanDocument<ILogicSchema>, Primitive>
+export type LogicDto = ILogic & { _id?: IFieldSchema['_id'][] }
```

Comment:
not that it makes a big difference, but why `IFieldSchema['_id']` instead of `ILogicSchema['_id']`?

Reply:
thanks! Actually I think the most correct one should be `Document['_id']`, since that is where all the Schemas inherit the `_id` property from